README.md

```diff
@@ -23,15 +19,42 @@ PHP client library for [Coveralls](https://coveralls.io).
 
 # Installation
 
+tl;dr
+Try the phar file first, come back if it doesn't work for you.
```

Comment:
Ah OK. I thought phar installation is something we should recommend, but I got your point.